admin/RebuildIndexesUsingCollations.pl

```diff
@@ -3,10 +3,35 @@
 use warnings;
 
 use FindBin;
+use Getopt::Long qw( GetOptions );
 use lib "$FindBin::Bin/../lib";
 use MusicBrainz::Server::Context;
+use aliased 'MusicBrainz::Server::DatabaseConnectionFactory' => 'Databases';
 
-my $c = MusicBrainz::Server::Context->create_script_context(database => 'MAINTENANCE');
+my $database = 'MAINTENANCE';
+my $concurrently = 1;
+my $show_help = 0;
+
+sub usage {
+    print <<~EOF;
+        Usage: RebuildIndexesUsingCollations.pl [options]
+
```

Comment:
The help message can contain a summary/reminder of the purpose of the script:
```suggestion
        Usage: RebuildIndexesUsingCollations.pl [options]

        Rebuild all indexes using collations,
        as needed in case of version mismatch after an update of glibc or libcu.

        Options:
```